src/ng1/viewDirective.ts

```diff
@@ -1,9 +1,18 @@
 /** @module view */ /** for typedoc */
-import {extend} from "../common/common";
+import {extend, map} from "../common/common";
 import {isDefined, isFunction} from "../common/predicates";
 import {trace} from "../common/trace";
-import {ViewConfig} from "../view/view";
-import {UIViewData} from "../view/interface";
+import {ActiveUIView} from "../view/interface";
+import {Ng1ViewConfig} from "./viewsBuilder";
+
+export type UIViewData = {
```

Comment:
This is now the shape of the data we place on the `$element.data('$uiView')`.   The `Ng1ViewConfig` holds most of the information the `ui-view` needs to become active, such as the fully realized `controller` and `template` properties.